src/atomate2/vasp/sets/defect.py

```diff
@@ -0,0 +1,59 @@
+"""Module defining VASP input set generators for defect calculations."""
+
+from dataclasses import dataclass
+
+from pymatgen.core import Structure
+from pymatgen.io.vasp import Outcar, Vasprun
+
+from atomate2.vasp.sets.base import VaspInputSetGenerator
+
+
+@dataclass
+class AtomicRelaxSetGenerator(VaspInputSetGenerator):
+    """Class to generate VASP atom-only relaxation input sets."""
+
+    def __post_init__(self):
+        """Initialize the class."""
+        super().__post_init__()
+        self.use_structure_charge = True
```

Comment:
You don't need to do all of this. Can just set the class attribute directory. E.g.,

```suggestion
        use_structure_charge: bool = True
```

Reply:
oops yeah this had other logic in it before.